src/GrpcRequestWrapper.php

```diff
@@ -134,11 +143,11 @@ private function handleResponse($response)
         if ($response instanceof PagedListResponse) {
             return $response->getPage()
                 ->getResponseObject()
-                ->serialize(new PhpArray());
+                ->serialize($this->codec);
```

Comment:
Minor style suggestion...

Since `$response->getPage()->getResponseObject()` returns an instance of `Message`, perhaps this would be a bit better?

````php
if ($response instanceof PagedListResponse) {
    $response = $response->getPage()->getResponseObject();
}

if ($response instanceof Message) {
    return $response->serialize($this->codec);
}
````